style.css

```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
```

Comment:
- [ ] Please try to be more professional in your comments as suggested by your previous code reviewer. Your comment seems like ok but you can still improve it and can write something like below.

### Example

Well done you did a great job by using the correct font family. Please look at the below points to understand how the `font-family` property helps to use different fonts of our own choice using the CDN.
   - Font family refers to a collection of fonts that share similar design characteristics, such as size, weight, and style. When specifying a font family in a web design, it is important to choose a font that is legible and appropriate for the content, style, and tone of the website.